project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.5.8
\ No newline at end of file
+sbt.version = 1.6.2
```

Comment:
why do you change sbt version?

Reply:
[info] [launcher] getting org.scala-sbt sbt 1.6.2  (this may take some time)...
[info] [launcher] getting Scala 2.12.15 (for sbt)...

Maybe instead of using scala 2.12.14 we can put scala 2.12.15 and it wouold solve the compilation errors?